explainer.md

```diff
@@ -0,0 +1,107 @@
+# Screen Wake Lock API
+
+## Introduction
+The **Screen Wake Lock API** allows web applications to prevent a device's
+screen from dimming and locking.
+
+See also:
+* [The current specification](https://w3c.github.io/screen-wake-lock/)
+* [web.dev article](https://web.dev/wakelock/) on the Screen Wake Lock API
+
+## Why?
+To avoid draining the battery, most devices quickly go to sleep when left idle.
+While this is fine most of the time, some applications need to keep the screen
+awake to complete their work. For example:
+
+* A recipe app that keeps the screen on while you bake a cake or cook dinner.
+* A boarding pass or ticket app that keeps the screen on until the barcode has
+  been scanned.
+* A kiosk-style app that keeps the screen on continuously.
+* A web-based presentation app that keeps the screen on during a presentation.
+
+Without this API, web developers have had to rely on libraries such as
+[NoSleep.js](https://github.com/richtr/NoSleep.js), which essentially added a
+[small
+video](https://github.com/richtr/NoSleep.js/blob/eaf52afd1dfbb80145b4a39f3ec29307b80ab154/src/index.js#L37-L57)
+to a page to keep the display on.
+
+> Recent NoSleep.js releases have added support for the Screen Wake Lock API.
+
+The purpose of this specification is to allow sites to be explicit about when
+they want to acquire a wake lock so that browsers can provide better automated
+and manual controls to prevent intentional and incidental issues.
+
+## Non-goals
+A previous iteration of this spec included both Screen and System wake locks.
+The latter has been moved to a separate spec, and is out of scope for this
+explainer.
```

Comment:
I don't think we need to mention this at all, tbh. 

```suggestion
```

Reply:
Why isn't this historical context useful? I think it's important for current specifications to reference previously tried ideas as "alternatives considered" or "non-goals" to avoid repeating old discussions endlessly.

Reply:
I was thinking we should have this in the spec, tbh.

Reply:
I think non-goals section is essential for this explainer given the historical context, see also https://github.com/w3ctag/w3ctag.github.io/blob/master/explainers.md#non-goals

That said, including similar text in the spec as well might also help readers who lack the historical context.